embedded-files/tekton/triggers.yaml

```diff
@@ -229,12 +266,74 @@ metadata:
 spec:
   params:
   - name: IMAGE-SHA
-  - name: RUN-ID
+  - name: MODEL-URI
   workspaces:
   - name: source
   steps:
   - name: run
     image: lachlanevenson/k8s-kubectl
     workingDir: "/workspace/source/app"
     script: |
-      sed "s/#RUN_ID#/$(params.RUN-ID)/g; s/#IMAGE_SHA#/$(params.IMAGE-SHA)/g" .fuseml/serve.yaml | kubectl apply -f -
+      #!/bin/sh
+      sed "s#__MODEL_URI__#$(params.MODEL-URI)#g; s#__IMAGE_SHA__#$(params.IMAGE-SHA)#g; s#__AWS_ACCESS_KEY_ID__#${AWS_ACCESS_KEY_ID}#g; s#__AWS_SECRET_ACCESS_KEY__#${AWS_SECRET_ACCESS_KEY}#g" .fuseml/serve.yaml | kubectl apply -f -
```

Comment:
just wondering - is it safe to use # delimiter for IMAGE_SHA and S3 access keys?

Reply:
I think its ok. SHA is based on hexadecimal and the s3 keys are alphanumeric.